src/core/pyproxy.ts

```diff
@@ -18,6 +18,7 @@ declare var Tests: any;
 declare var Module: any;
 declare var Hiwire: any;
 declare var API: any;
+declare function sleep(ms: number): Promise<undefined>;
```

Comment:
I put it in `pre.js` because that makes it visible everywhere (except in the loader logic): inside of `api.ts`, inside of `EM_JS`, etc. `sleep` is a kind of simple utility function, so I figure it could eventually be useful in other places. But it could be in `pyproxy.ts`, it's not all that important where it goes.